addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.xml

```diff
@@ -21,28 +21,21 @@
                 <sheet>
                     <group>
                         <group id="header_left_group">
-                            <field name="date"/>
-                            <field name="journal_id" domain="[('type', '=', 'general')]"/>
+                            <field name="date" string="Entry Date"/>
+                            <field name="base" widget="monetary" options="{'currency_field': 'currency_id'}" required="True"/>
+                            <field name="tax_id" domain="[('l10n_in_tds_tax_type', '=', l10n_in_tds_tax_type)]" required="True"/>
+                            <field name="amount"/>
```

Comment:
Why not monetary?  Or why do we need monetary in L25?

Reply:
Okay will make amount monetary.